actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -47,10 +44,10 @@ class ActionCable.Connection
   isOpen: ->
     @isState("open")
 
-  # Private
+  isActive: ->
+    @isState("open", "connecting")
```

Comment:
Renamed `isAlive` to `isActive` (and inverted the state condition) because I feel it's a more accurate description. Functionally, the methods are the same.